modules/core/src/lib/effect-manager.js

```diff
@@ -1,4 +1,5 @@
 import {deepEqual} from '../utils/deep-equal';
+import {default as LightingEffect} from '../effects/lighting-effect';
```

Comment:
I think it would be better that the instantiator of the EffectManager (i.e. `Deck`) would choose to add this default `Effect`, rather than the `EffectManager` always including it.

Reply:
The setProps in Deck is already pretty messy, better only manage things in high level, that's why I preferred to avoid extra effect checking and adding logic in Deck.